estate/views/estate_menu_views.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <menuitem
+        id="ep_menu_root"
+        name="Real Estate"
+        sequence="6"/>
+    <menuitem
+        id="ep_l1_adv_menu"
+        name="Advertisements"
+        parent="ep_menu_root"/>
+    <menuitem
+        id="ep_model_menu_action"
+        action="act_estate_property"
+        parent="ep_l1_adv_menu"/>
+    <menuitem
+        id="ep_l1_settings_menu"
+        name="Settings"
+        parent="ep_menu_root"/>
+    <menuitem
+        id="ep_type_model_menu_action"
+        action="act_estate_property_type"
+        parent="ep_l1_settings_menu"
+        sequence="1"/>
+    <menuitem
+        id="ep_tag_model_menu_action"
+        action="act_estate_property_tag"
+        parent="ep_l1_settings_menu"
+        sequence="2"/>
+</odoo>
```

Comment:
```suggestion
<odoo>
    <menuitem id="real_estate_menu_root" name="Real Estate">
        <menuitem id="real_estate_adv_menu" name="Advertisements">
            <menuitem id="estate_property_model_menu_action" action="act_estate_property" />
        </menuitem>

        <menuitem id="real_estate_settings_menu" name="Settings">
            <menuitem id="estate_property_type_model_menu_action" action="act_estate_property_type" />
            <menuitem id="estate_property_tag_model_menu_action" action="act_estate_property_tag" />
        </menuitem>
    </menuitem>
</odoo>
```
When menuitems are related and defined in the same file, you can nest them for improved readability.
Also, same remark as before about explicit ids.